CHANGELOG.md

```diff
@@ -55,6 +55,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 
 - Correct the release date of `0.20.0` to `2024-04-25`.
+- Fixed `Int` type precision error that occurs when Graphql API have instances
+  that are greater than `i32` (`u32`,`u64`,`isize`,`usize`,`i64`) for
+  `blockListLdap`,`ldapPlainText`,`ldapBruteForce`,`blockListKerberos`,
+  `HttpThreat`,`torConnection`,`domainGenerationAlgorithm`,`nonBrowser`,
+  `blockListHttp`,`extraThreat`.
```

Comment:
To maintain consistency with `HttpThreat`, it would be better to use capital characters like `BlockListLdap` instead of `blockListLdap`. What do you think?

Reply:
Sounds good. Will do 👍